### PR TITLE
Content update

### PR DESCRIPTION
**Pull request by ska-ibees**
MultiCloud-Native changed to MultiCloud Native 
License info removed

### CODE REVIEW

**Comment by ska-ibees** [comment]
@JuanDCG This PR created within seperate branch. Let me know your thoughts.

File: _pages/terradoocloud.md [hunk 1]
```diff
@@ -6,65 +6,68 @@ permalink: /
 {{ content }}
 
 
-What Is ?
+
+
+ 
+![Terradoo Cloud](source/_themes/images/Terradoo-cloud-new-logo.png)
```
Comment (JuanDCG) [inline _pages/terradoocloud.md:12]: Remove this line as a separate PR

**Comment by JuanDCG** [comment]
Could you please remove the image references to merge this PR as only content update ?

**Comment by ska-ibees** [comment]
> 
> 
> Could you please remove the image references to merge this PR as only content update ?

Done in #4

**Comment by JuanDCG** [comment]
Ok, thank you, the last thing should be delete the new-logo image from assets/ to only merge the Content update.

`assets/images/Terradoo-new-logo.png`

**Pull request closed without merging**